### ISSUE TITLE
Improve visibility of mod panel actions when in dark mode

### ISSUE THREAD

**Issue by maestromac** [forem/forem#18683]
### Discussed in https://github.com/forem/forem/discussions/18579

<div type='discussions-op-text'>

<sup>Originally posted by **michael-tharrington** October 13, 2022</sup>
It was recently brought up by a couple of DEV moderators that it's really difficult to see your selections when trying to add/remove a tag from someone's post.

Here's a screenshot that illustrates the issue ("discuss" is being selected in the image):

![Screen Shot 2022-10-13 at 9 13 10 AM](https://user-images.githubusercontent.com/16007075/195606332-f883aaf9-3608-466c-99f8-a18b64d7aeaa.png)

We may want to look into button selections across the board in dark mode. There are a couple other similar instances to this one that have been flagged:
- https://github.com/forem/forem/discussions/17896
- https://github.com/forem/forem/discussions/18305</div>

**Comment by mirie** [comment]
@anujbhavsar96 this is being tracked as a bug. Do you have recommendations for how this should look in dark mode?

**Comment by anujbhavsar96** [comment]
I feel tweaking an active state CSS a bit should solve this. For example, we can use `background: var(--base-10)` for dark and light modes. @mirie 

<img width="902" alt="Screenshot 2022-11-08 at 10 41 44 AM" src="https://user-images.githubusercontent.com/84626054/200480706-8fa45d68-763e-4c84-bfd0-ab71677e44d0.png">

**Comment by rt4914** [comment]
@mirie Assigning this to myself as while investing this, it was a quick solution. 

@anujbhavsar96 Can you also have a look at other two discussions mentioned in description, so that we can fix them too?

Also, @mirie @anujbhavsar96 you can have a look at final output in this PR description https://github.com/forem/forem/pull/18764
If any changes are needed, do let me know.

**Comment by anujbhavsar96** [comment]
> Make the "I agree" signup check boxes more visible

@rt4914  For this just changing the `--border-width`  to `1.5px` should be enough.

<img width="337" alt="Screenshot 2022-11-24 at 10 32 35 AM" src="https://user-images.githubusercontent.com/84626054/203698681-9a33946b-1dd5-48ab-82c1-35fbec46dff7.png">

**Comment by anujbhavsar96** [comment]
> Improve inline code preview in dark mode when on draft

@rt4914 ideal solution here looks like below, where we change the `background` to `rgba(255,255,255,0.1)` for the code class. But I think that affects the Code block component as well. 

![canary1 forem lol_new](https://user-images.githubusercontent.com/84626054/203710959-d916d6a7-9084-4b8d-9ddd-9199d6de35e4.png)

**Comment by rt4914** [comment]
@anujbhavsar96 I have applied your suggested changes here https://github.com/forem/forem/pull/18785
